### PR TITLE
Add script to read Java Card details

### PR DESCRIPTION
**Pull request by arsalansufi**
_Easiest reviewed by commit_

## Overview

This PR adds a script to read Java Card details, namely environment, jurisdiction, user role, and election hash. This should make troubleshooting issues with Java Cards, like mismatched environments or jurisdictions, much easier!

**Usage**

```
$ ./scripts/read-java-card-details 
```

**Sample outputs**

_No card or card reader_

```
❌ Card status not "ready" after 3 seconds
```

_Card has not been run through initial Java Card configuration script_

```
Env:           -
Jurisdiction:  -
User role:     -
Election hash: -
```

_Card has been run through initial Java Card configuration script but not programmed for a user_

```
Env:           production
Jurisdiction:  -
User role:     -
Election hash: -
```

_Card has been run through initial Java Card configuration script and programmed for a user_

```
Env:           development
Jurisdiction:  st.dev-jurisdiction
User role:     system_administrator
Election hash: -
```

```
Env:           development
Jurisdiction:  st.dev-jurisdiction
User role:     election_manager
Election hash: 00d11395693096748e8415eb5ff530ae09c87c28deb669ce1ab98100bbe4dad3
```

## Testing

- [x] Tested script manually
- [x] Added tests to maintain 100% code coverage

### CODE REVIEW

**Comment by adghayes** [comment]
@kshen0 heads up, beginning to review this one, no need on your part.

**Comment by kshen0** [comment]
Thanks @adghayes!

File: libs/auth/src/card_reader.ts [hunk 1]
```diff
@@ -97,6 +100,15 @@ export class CardReader {
     });
   }
 
+  /**
+   * Disconnects the currently connected card, if any
```
Comment (adghayes) [inline libs/auth/src/card_reader.ts:104]: "connected card" or "connected card reader"?

File: libs/auth/src/java_card.ts [hunk 2]
```diff
@@ -702,9 +702,39 @@ export class JavaCard implements Card {
     );
   }
 
+  //
+  // Methods for scripts
+  //
+
+  /**
+   * Disconnects the card so that it can be reconnected to, through a new JavaCard instance
```
Comment (adghayes) [inline libs/auth/src/java_card.ts:710]: same comment question

**Review by adghayes — APPROVED** [review]
LGTM

File: libs/auth/src/card_reader.ts (hunk 1 above)
Comment (arsalansufi) [reply libs/auth/src/card_reader.ts:106]: Updated the naming scheme such that it's clearer at both the `CardReader` and `JavaCard` levels that we're talking about disconnecting the card and not the card reader:
- `CardReader.disconnectCard`
- `JavaCard.disconnect`

**Pull request merged**